### ISSUE TITLE
"a UserDefinedError occurred (error 2318), reason:halt, No loaded method for #'_____could_not_parse_selector_from_method_source_____' was found for class 'RwPrjLoadTool'"

### ISSUE THREAD

**Issue by dalehenrich** [dalehenrich/Rowan#77]
Getting the above error when compiling methods with a leading $_  ... here's a snapshot of the stack:
![image](https://user-images.githubusercontent.com/623951/38956960-062ac402-430e-11e8-8b16-c42574f04346.png)

The methodDef seems to be coming from:
```smalltalk
RwMethodDefinition newForSource: methodSource protocol: protocol.
```
but the selector `_____could_not_parse_selector_from_method_source_____` does not ring a bell and oddly it does not seem to affect the compiliation/addition of the original method.

I'm in the middle of something else, so I'll track this down later ...

**Comment by dalehenrich** [comment]
I see that RwGsPlatform>>parseSelectorFrom: returns the `_____could_not_parse_selector_from_method_source_____` selector to indicate an error in parsing the method selector ...

**Comment by dalehenrich** [comment]
fix will be available in 0.0.4 ...
